entsoe-util/src/main/java/com/powsybl/entsoe/util/EntsoeFileName.java

```diff
@@ -77,7 +77,11 @@ public EntsoeGeographicalCode getGeographicalCode() {
     }
 
     public String getCountry() {
-        return geographicalCode != null ? geographicalCode.getCountry().toString() : null;
```

Comment:
Perhaps a more dense implementation could be better ?
```java
return geographicalCode != null && geographicalCode.getCountry() != null ? geographicalCode.getCountry().toString() : null;
```
@mathbagu what do you think about it ?

Reply:
Agree: @amirakahya could you change the code as @murgeyseb suggest.

For you information, it's not necessary to prefix geographicalCode with `this` like you did. Use this syntax only if there is a clash between a variable and an attribute of the class. For instance, it can happen in a setter:
```
public void setX(int x) {
    this.x = x;
}
```